README.md

```diff
@@ -37,6 +37,28 @@ to your Github account:
 
 You should now receive an email whenever the changelog is updated.
 ---
+### 3 July 2023
+The following changes are now available in production:
+
+### individuals-charges-api
+
+New API Version V2.0 is now available
+
+ * `Retrieve Pensions Charges` and `Create and Amend Pensions Charges` have been updated with `isAnnualAllowanceReduced`, `moneyPurchasedAllowance` and `taperedAnnualAllowance` removed from  `pensionSavingsTaxCharges` and added to  `pensionContributions`
+
+All endpoints replace their respective v1 equivalents, which are now deprecated. Please use the new v2 endpoints instead.
+### individuals-calculations-api
+
+New API version V4.0 is now available in production
+  * `List Self Assessment Tax Calculations` has been updated with, `biss` and `POA` removed from the `calculationType`, `totalIncomeTaxAndNicsDue` is now optional and
```

Comment:
change to:

`List Self Assessment Tax Calculations` has been updated: `biss` and `POA` are removed from the `calculationType`, `totalIncomeTaxAndNicsDue` is now optional and